homeassistant/components/verisure/alarm_control_panel.py

```diff
@@ -55,33 +55,47 @@ def unique_id(self) -> str:
         """Return the unique ID for this entity."""
         return self.coordinator.entry.data[CONF_GIID]
 
-    async def _async_set_arm_state(self, state: str, code: str | None = None) -> None:
+    async def _async_set_arm_state(self, state: str, command_data: dict) -> None:
```

Comment:
Would it help if the query functions returned a Verisure.Session.Query instead of dict?

https://github.com/persandstrom/python-verisure/pull/163/files